src/core/js/modelEvent.js

```diff
@@ -0,0 +1,32 @@
+define(function() {
+
+    var ModelEvent = Backbone.Controller.extend({
+
+        type: null,
+        target: null,
+        value: null,
+        bubbles: true,
+        deepPath: null,
+        target: null,
+        timeStamp: null,
+
+        initialize: function(type, target, value) {
```

Comment:
a JSDoc comment would be useful here to document the function parameter data types at least